src/primitives/transaction.h

```diff
@@ -10,8 +10,10 @@
 #include "script/script.h"
 #include "serialize.h"
 #include "uint256.h"
+#include "pubkey.h"
 
 static const int SERIALIZE_TRANSACTION_NO_WITNESS = 0x40000000;
+static const int PSBT_MAGIC_BYTES = 0x74627370; // Note this is reversed from the actual magic bytes sequence just for the seralizer.
```

Comment:
You could mention it being reserved due to serialization being in LSB order.

Also, this can probably move to script/sign.